app/uploaders/court_images_uploader.rb

```diff
@@ -2,19 +2,17 @@
 
 class CourtImagesUploader < CarrierWave::Uploader::Base
 
-  # Include RMagick or MiniMagick support:
   include CarrierWave::RMagick
-  # include CarrierWave::MiniMagick
 
-  # Include the Sprockets helpers for Rails 3.1+ asset pipeline compatibility:
-  # include Sprockets::Helpers::RailsHelper
-  # include Sprockets::Helpers::IsolatedHelper
-
-  # Choose what kind of storage to use for this uploader:
-  #storage :fog
+  if Rails.env.development? || Rails.env.test?
```

Comment:
Yes about this one. Ideally, I would like to replace this with ENV variable so there will be no logic included. I'm waiting for guys to show me/tell me how to add ENV variable to our servers.